app-laptop/laptop-mode-tools/laptop-mode-tools-1.74.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit multilib systemd udev
+
+MY_P="${PN}_${PV}"
+
+DESCRIPTION="Linux kernel laptop_mode user-space utilities"
+HOMEPAGE="https://github.com/rickysarraf/laptop-mode-tools/wiki"
+SRC_URI="https://github.com/rickysarraf/${PN}/releases/download/${PV}/${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 ppc x86"
+IUSE="+acpi apm bluetooth"
+
+RDEPEND="sys-apps/iproute2
+	sys-apps/ethtool
+	sys-apps/which
+	|| (
+		sys-apps/sdparm
+		sys-apps/hdparm
+	)
+	acpi? ( sys-power/acpid )
+	apm? ( sys-apps/apmd )
+	bluetooth? ( net-wireless/bluez:= )"
+
+S="${WORKDIR}/${MY_P}"
+
+src_compile() { :; }
+
+src_install() {
+	DESTDIR="${D}" \
+		INIT_D="none" \
+		MAN_D="/usr/share/man" \
+		LIB_D="/usr/$(get_libdir)" \
+		UDEV_D="$(get_udevdir)" \
+		SYSTEMD_UNIT_D="$(systemd_get_systemunitdir)" \
+		TMPFILES_D="/usr/lib/tmpfiles.d" \
+		ACPI="$(use acpi && echo force || echo disabled)" \
+		PMU="$(false && echo force || echo disabled)" \
+		APM="$(use apm && echo force || echo disabled)" \
+		SYSTEMD=yes \
+		sh ./install.sh || die
+
+	dodoc Documentation/*.txt README.md
+	newinitd "${FILESDIR}"/laptop_mode.init-1.4 laptop_mode
+
+	rm "${D}/usr/lib/pm-utils/sleep.d/01laptop-mode"
+}
+
+pkg_postinst() {
+	if use acpi || use apm; then
+		if use acpi; then
+			daemon_name="acpid"
+		elif use apm; then
+			deamon_name="apmd"
+		fi
+		if [ "$(rc-config list default | grep laptop_mode)" = "" ] || [ "$(rc-config list default | grep ${daemon_name} )" = "" ]; then
```

Comment:
This looks a bit odd and I guess assumes OpenRC? Do we need this?

Reply:
I think I'll just print the message the first time automatically. 
There's some functions to print a messages only the first time a package is installed, but I can't find it now. Do you know how to do this with portage?

Reply:
Something like [this](https://gitweb.gentoo.org/repo/gentoo.git/tree/net-irc/ngircd/ngircd-26.ebuild#n102) should work. This [list](https://devmanual.gentoo.org/ebuild-writing/variables/) is always helpful too.

Reply:
Done!